src/main/starlark/builtins_bzl/common/cc/attrs.bzl

```
@@ -119,10 +119,36 @@ most build rules</a>.
    and that rule's <code>outs</code> are automatically added to
    this <code>cc_library</code>'s data files.
 </p>
-<p>Your C++ code can access these data files like so:</p>
+<p>Using the functionality defined by the <code>runfiles_src.h</code> located under 
+   <code>tools/cpp/runfiles/runfiles_src.h</code> Your C++ code can access these data files,
+   like so:
+</p>
 <pre><code class="lang-starlark">
-  const std::string path = devtools_build::GetDataDependencyFilepath(
-      "my/test/data/file");
+  
+  #include "runfiles_src.h"
+
+  int main(int argc, char** argv) {
+    
+    try {
+
+    using bazel::tools::cpp::runfiles::Runfiles;
+    
+    std::string error;
+    std::unique_ptr<Runfiles> runfiles(Runfiles::Create(argv[0], BAZEL_CURRENT_REPOSITORY, &error));
+    
+    // "cpp_example" is the WORKSPACE name attribute
```

```suggestion
    // "cpp_example" is the module name attribute
```
WORKSPACE is deprecated, we should not add new references to it.